AWSTranscribeStreaming/Internal/AWSTranscribeStreamingWebSocketProvider.h

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+#import "AWSTranscribeStreamingWebSocketProviderDelegate.h"
+#import "AWSSRWebSocketDelegateAdaptor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol AWSTranscribeStreamingWebSocketProvider <NSObject>
+
+@required
+-(void)send:(id)message;
+-(void)connect;
+-(void)disconnect;
```

Comment:
any particular reason why we went with connect/disconnect vs open/close?

Reply:
nope just from reading and looking at other code, I think I did connect and disconnect but I can change it